src/screens/EmailForm/index.tsx

```diff
@@ -40,18 +39,22 @@ const EmailForm = ({ route, navigation }: EmailFormProps) => {
     return () => { BackHandler.removeEventListener('hardwareBackPress', hardwareBackPress); };
   }, [hardwareBackPress]);
 
+  useEffect(() => {
+    if (Platform.OS === 'ios') setBehavior('padding');
+    else setBehavior('height');
+  }, []);
```

Comment:
Que penses-tu de : ` useEffect(() => { Platform.OS === 'ios' ? setBehavior('padding') : setBehavior('height') }, []);` ?

on pourrait aussi faire `useEffect(() => { setBehavior(Platform.OS === 'ios' ? 'padding' : 'height') }, []);`, mais je trouve ça moins compréhensible

Reply:
ta premiere proposition ne fonctionne que dans le cas d'un retour de fonction ou dans un cas comem ta 2eme ecriture (retour de valeur)... 
je l'avais d'abord ecrit comme ca et ca m'a mis une erreur lint
`Expected an assignment or function call and instead saw an expression.eslintno-unused-expressions`

Reply:
Ah oui, tout à fait !